sdk/mongocluster/arm-mongocluster/test/public/mongoclusterOperations.spec.ts

```diff
@@ -63,18 +63,15 @@ describe("MongoCluster test", () => {
       {
         location,
```

Comment:
remove autorest lisense https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/mongocluster/arm-mongocluster/test/public/node/privateEndpointConnections.spec.ts#L5-L6